components/raftstore/src/store/worker/read.rs

```diff
@@ -390,7 +396,7 @@ where
     E: KvEngine,
 {
     pub fn new(kv_engine: E, store_meta: Arc<Mutex<StoreMeta>>, router: C) -> Self {
-        let cache_read_id = ThreadReadId::new();
+        let cache_read_id = ThreadReadId::default();
```

Comment:
I mean this part. Does clippy requires the use of “default” instead of “new”?

Reply:
No, this have nothing to do with clippy requirement, If there is no need to make any changes here , i will restore the logic